src/wp-includes/widgets.php

```diff
@@ -2105,3 +2105,29 @@ function wp_check_widget_editor_deps() {
 		}
 	}
 }
+
+/**
+ * Register the previous theme's sidebars for the block themes.
```

Comment:
Nitpick: Use third-person forms in docblock shortdesc: "Register**s**".

Reply:
Done ✅